### PR TITLE
Bump Apache Avro to 1.9.2

### PR DESCRIPTION
**Pull request by Fokko**
This minor version bump fixes regression issues:
https://github.com/apache/avro/releases/tag/release-1.9.2

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>98</version>
+        <version>99</version>
```
Comment (ebyhr) [inline pom.xml:8]: The latest airbase is 98. https://github.com/airlift/airbase/releases

Comment (Fokko) [reply pom.xml:8]: Thanks for pointing out, in the changes 99 is already mentioned, so I was under the impression that it was already released.

**Comment by Fokko** [comment]
Rebased onto master

**Comment by electrum** [comment]
Please squash commits so that we don’t have a broken commit in the middle. You could also flip the commit order.

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**